src/main/java/pages/HomePageFS.java

```diff
@@ -0,0 +1,22 @@
+package pages;
+
+import chunks.GalleryOverlay;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePageFS {
+
+    @FindBy(xpath = "//div[@class='Gallery Hero-gallery']/button[@class='Button--underlined--dark Gallery-button']/span[@class='Button-label']")
```

Comment:
Add a block comment for the element getters such as the one for methods: //Private Elements

Reply:
Done